aicsimageio/readers/tiff_reader.py

```diff
@@ -224,23 +212,39 @@ def _read_delayed(self) -> xr.DataArray:
                 # Convert the numpy array of lazy readers into a dask array
                 image_data = da.block(lazy_arrays.tolist())
 
-                # Get metadata from tags
-                metadata = self._get_metadata(
-                    fs=self.fs,
-                    path=self.path,
-                    scene=self.current_scene,
-                )
+                return image_data
 
-                # Create dims and coords
-                dims = self._guess_dim_order()
-                coords = self._get_coords(dims, image_data.shape)
+    def _read_delayed(self) -> xr.DataArray:
+        """
+        Construct the delayed xarray DataArray object for the image.
 
-                return xr.DataArray(
-                    image_data,
-                    dims=dims,
-                    coords=coords,
-                    attrs=metadata,
-                )
+        Returns
+        -------
+        image: xr.DataArray
+            The fully constructed and fully delayed image as a DataArray object.
+            Metadata is attached in some cases as coords, dims, and attrs.
+
+        Raises
+        ------
+        exceptions.UnsupportedFileFormatError: The file could not be read or is not
+            supported.
+        """
+        # Create the delayed dask array
+        image_data = self._create_dask_array()
+
+        # Get unprocessed metadata from tags
+        tiff_tags = self._get_tiff_tags()
+
+        # Create dims and coords
+        dims = self._guess_dim_order()
+        coords = self._get_coords(dims, image_data.shape)
+
+        return xr.DataArray(
+            image_data,
+            dims=dims,
+            coords=coords,
+            attrs={constants.METADATA_UNPROCESSED: tiff_tags},
```

Comment:
all the tiff tags?  excessive?

Reply:
This is the `TiffReader` not `OmeTiffReader` though. And even then, why _not_.

Reply:
1. imagedescription is not only used by ome-tiffs.  imagej and other tiff writing apps stash their own metadata in there. 
2. The why not is that it's potentially a lot of noise that is very tiff-specific (including the tag names).  It's not what we normally consider metadata.  Dims have already been extracted in other ways.   If this surfaces to the end user you have to forward them to the TIFF file format spec.   I'm not going to argue any more strongly, I can't deny that it provides good info about the image but it's pretty low level.

Reply:
That looks interesting to me.  Could `img.metadata` become the ome-types object in the case of ome-tiff, or am I at the wrong layer for that?

Reply:
That is the current behavior for the `OmeTiffReader` `img.metadata` -> `OME` object.

This is the `TiffReader` so just str for this one?

Reply:
sounds good!